akka-actor/src/main/resources/reference.conf

```diff
@@ -276,6 +276,9 @@ akka {
 
         # Max number of threads to cap factor-based parallelism number to
         parallelism-max = 64
+
+        # Enable establishing local first-in-first-out scheduling mode for forked tasks that are never joined.
+        async-mode = on
```

Comment:
so just FIFO or LIFO?I don't how to describe that in a proper way.could you please just comment here so I could copy and paste?